lib/oregon_digital/controlled_vocabularies/extended_location.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module OregonDigital
+  module ControlledVocabularies
+    # Ethnographic Term object for storing labels and uris
+    class ExtendedLocation < Resource
+      def self.all_endpoints
+        [
+          OregonDigital::ControlledVocabularies::Vocabularies::OnsTDFFBasin
```

Comment:
Is this supposed to be TDFF or TFDD. The file controlled_vocabularies/vocabulaties/ons_tdffbasin.rb has it as tdff but the uri used is tfdd. These should be synced up properly id assume.